### PR TITLE
Remove the cenc signals and use the fluc API instead

### PR DESCRIPTION
**Pull request by jbouzas**

### CODE REVIEW

File: gst/isomp4/qtdemux.c [hunk 1]
```diff
@@ -3050,6 +2965,11 @@ qtdemux_parse_moof (GstQTDemux * qtdemux, const guint8 * buffer, guint length,
   g_node_destroy (moof_node);
   return TRUE;
 
+cenc_parse_error:
```
Comment (aslobodeniuk) [inline gst/isomp4/qtdemux.c]: label cenc_parse_error is unused (if we don't enable fluc-sdk), on new versions of gcc compilation will fail

File: gst/isomp4/qtdemux.c [hunk 2]
```diff
@@ -5209,16 +5118,12 @@ qtdemux_parse_moov (GstQTDemux * qtdemux, const guint8 * buffer, guint length)
...
   if (pssh) {
-    GstBuffer *buf;
-
-    /* for the pssh, just send the buffer completely */
-    buf = gst_buffer_new ();
-    GST_BUFFER_DATA (buf) = pssh->data;
-    GST_BUFFER_SIZE (buf) = GST_READ_UINT32_BE (pssh->data);
-    g_signal_emit (qtdemux, gst_qtdemux_signals[SIGNAL_PSSH], 0, buf);
-    gst_buffer_unref (buf);
+    GstEvent *event = fluc_drm_event_new_pssh (pssh->data, "isobmff/moov");
```
Comment (aslobodeniuk) [inline gst/isomp4/qtdemux.c:5132]: fluc_drm_event_new_pssh can return NULL

Comment (jbouzas) [reply gst/isomp4/qtdemux.c:5132]: I will print an error if the event could not be created

Comment (jbouzas) [reply gst/isomp4/qtdemux.c:5132]: Done

**Pull request closed without merging**